Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/bugreporting/event/BugEvent.kt

```diff
@@ -0,0 +1,20 @@
+package de.rki.coronawarnapp.bugreporting.event
+
+import org.joda.time.Instant
+
+interface BugEvent {
+    val id: Long
+    val createdAt: Instant
+    val tag: String?
+    val info: String?
+    val exceptionClass: String
+    val exceptionMessage: String?
+    val stackTrace: String
+    val appVersionName: String
+    val appVersionCode: Int
```

Comment:
versionCode is offically int on my knowledge. As BuildConfig.VerionCode is also int, I would like to stay on Int in this case.

Reply:
Please use long:
https://developer.android.com/reference/android/content/pm/PackageInfo#versionCode

Int vs Long storage gain does not matter here, and changing this to `Long` is an easy thing we can do to prevent future headaches. Casting it now is a lot easier than later changing database schemas

Reply:
done.